app/preview.py

```diff
@@ -225,7 +217,7 @@ def get_html(json, language="english"):
 
 
 @sentry_sdk.trace
```

Comment:
I wonder - do we still use this sentry tracing? Doesn't have to be removed in this PR, but might be a good time to think if we still need it?

Reply:
hmmm i think lets keep this in here - i think i'd like a wider conversation about sentry before we decide what to do with these 🤔

Reply:
Okee 👍🏼